airflow/auth/managers/simple/ui/src/login/Login.tsx

```diff
@@ -36,7 +36,7 @@ type ExpandedApiError = {
 export const Login = () => {
     const onSuccess = (data: LoginResponse) => {
         // Redirect to index page with the token
-        globalThis.location.replace(`/webapp/?token=${data.jwt_token}`);
+        globalThis.location.replace(`/?token=${data.jwt_token}`);
```

Comment:
(@pierrejeambrun not going to worry about it in this PR, but this doesn't work with a path in `base_url`. I was able to easily fix the other auth managers, but not this one.)

Reply:
I also have a question about why we are putting the JWT token in a query param, which means it gets sent and shows up in server access logs.

Reply:
Fair question! Not going to block this PR on it though.

Reply:
I opened #47080 so we don't forget.